embeddings/pipeline/lightning_hps_pipeline.py

```diff
@@ -178,3 +161,55 @@ def _pop_sampled_parameters(
             task_train_kwargs,
             model_config_kwargs,
         )
+
+
+@dataclass
+class OptimizedLightingClassificationPipeline(OptimizedLightingPipeline):
+    def __post_init__(self) -> None:
+        # Type: ignore is temporal solution due to issue #152 https://github.com/CLARIN-PL/embeddings/issues/152
+        super(OptimizedLightingPipeline, self).__init__(
+            preprocessing_pipeline=HuggingFacePreprocessingPipeline(
+                dataset_name=self.dataset_name,
+                persist_path=self.tmp_dataset_dir.name,
+                sample_missing_splits=(self.sample_dev_split_fraction, None),
+                ignore_test_subset=True,
+                load_dataset_kwargs=self.load_dataset_kwargs,
+            ),
+            evaluation_pipeline=LightningClassificationPipeline,  # type: ignore
+            pruner=self.pruner_cls(n_warmup_steps=self.n_warmup_steps),
+            sampler=self.sampler_cls(seed=self.seed),
+            n_trials=self.n_trials,
+            dataset_path=self.tmp_dataset_dir.name,
+            metric_name="f1__average_macro",
+            metric_key="f1",
+            config_space=self.config_space,
+        )
+
+
+@dataclass
+class OptimizedLightingSequenceLabelingPipeline(OptimizedLightingPipeline):
```

Comment:
This Class has incorrect Metadata, because SequenceLabelingPipeline parameters are not covered by          metadata `LightningPipelineMetadata`